pkg/setting/expanders.go

```diff
@@ -138,7 +139,7 @@ func (e fileExpander) Expand(s string) (string, error) {
 		return "", err
 	}
 
-	f, err := ioutil.ReadFile(s)
+	f, err := ioutil.ReadFile(filepath.Clean(s))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.